### DISCUSSION TITLE
connect reactflow

### DISCUSSION THREAD

**Discussion by lexoring** [xyflow/xyflow#2697] [Q&A]
Hello. Tell me, please, is it possible to connect react-flow without node.js? I just haven't worked with react and node.js and don't understand some of the processes.
I thought that it is possible through CDN
```
<script crossorigin src="https://unpkg.com/react@18/umd/react.development.js"></script>
<script crossorigin src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
<script src="https://cdn.jsdelivr.net/npm/reactflow@11.3.3/dist/umd/index.min.js"></script>
```

then I create a reactflow.js file
and add text
`import ReactFlow, { Controls, Background } from 'reactflow';`

but in the browser this line shows an error.

**Comment by lexoring** [comment]
I figured everything out, it can be removed

**Reply by moklick** [reply]
Hey. Can you share your solution?

**Reply by lexoring** [reply]
I just decided to use the packages through npm. It's easier.
